thrift0.13/src/main/java/com/linecorp/armeria/server/thrift/THttpService.java

```diff
@@ -591,6 +591,7 @@ private void handleException(ServiceRequestContext ctx, CompletableFuture<HttpRe
         final RpcResponse response = handleException(ctx, cause);
         response.handle((result, convertedCause) -> {
             if (convertedCause != null) {
+                convertedCause = Exceptions.peel(convertedCause);
```

Comment:
What I meant was doing this at line `591`. 😅

Reply:
Oops sorry, updated 😅